tests/functional/Extension/HeadingPermalink/HeadingPermalinkExtensionTest.php

```diff
@@ -61,6 +61,7 @@ public function testHeadingPermalinksWithCustomOptions(string $input, string $ex
                 'symbol'          => '¶ 🦄️ <3 You',
                 'insert'          => 'after',
                 'title'           => 'Link',
+                'aria_hidden'     => false
```

Comment:
If you could please add a trailing comma (`,`) here everything else looks good to me! Nicely done :)

Reply:
Funny enough - thought I already had. Sorry for the double-commit and push. Hope it updates this time.